home_assistant_streamdeck_yaml.py

```diff
@@ -544,6 +823,35 @@ class Page(BaseModel):
         description="A list of buttons on the page.",
     )
 
+    dials: list[Dial] = Field(
+        default_factory=list,
+        description="A list of dials on the page."
+    )
+    
+    _dials_sorted: list[Dial] = PrivateAttr([])
+    def sort_dials(self) -> list[(Dial,Dial | None)]:
+        self._dials_sorted = []
+        last_num = -1
+        for i in range(0,len(self.dials)):
+            if i == last_num: continue
+            try:
```

Comment:
I am not sure why you are using a try/except here, is there no better way to check for the index?